### DISCUSSION TITLE
Looking for advice on developing cross-platforms apps from one codebase

### DISCUSSION THREAD

**Discussion by FreedomFightDev** [necolas/react-native-web#2117] [Help]
Hi, all! I am currently working on integrating RNW with an existing react-native app. I've ran into a few odd issues. Have any of you worked with a production-level app that uses one codebase for RN IOS, Android and RNW? I'm having great success building with RNW on it's own, but trying to get more info from some who might have tried working with web and mobile in one codebase.

**Comment by korziee** [comment]
Bit late to the table with this one, but https://github.com/devhubapp/devhub is a good example.

**Comment by FreedomFightDev** [comment]
Thank you very much!

Aaron

**Reply by riteshoyo** [reply]
@FreedomFightDev I am also working on the common codebase and wanted to discuss how're you handling the testcases in common codebase?
